RELEASES.md

```diff
@@ -48,4 +48,5 @@ Specifications have a variety of different timelines in their lifecycle.
   For example if a breaking change is introduced in v1.0.0-rc2 then the series would end with v1.0.0-rc4 and v1.0.0.
 - Minor and patch releases SHOULD be made on an as-needed basis.
 
+
```

Comment:
Same as [here][1], I'd rather drop this (and have it come through [upstream][2] if you feel strongly about it).

[1]: https://github.com/opencontainers/runtime-spec/pull/687/#discussion_r104210348
[2]: https://github.com/opencontainers/project-template